roles/federation-registry/defaults/main.yml

```diff
@@ -0,0 +1,30 @@
+---
+# defaults file for federation-demo
+federation_registry_key_title: "evironment-vm"
+federation_registry_git:
+  github_access_token: "{{vault_GITHUB_ACCESS_TOKEN}}"
+  key_path: "~/.ssh/id_rsa.git"
+  git_repo: "git@github.com:linathedog/rciam-federation-registry.git"
+  git_branch: "devel"
+  clone_dest: "/var/www/rciam-federation-registry"
+federation_registry_known_hosts_path: "~/.ssh/known_hosts"
+federation_registry_db:
+  name: service_registry_db
+  username: postgres
+  password: postgres
```

Comment:
I suggest you do not use easy usernames and passwords.
I also suggest encrypting these variables with the Ansible vault.